WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsFragment.java

```diff
@@ -1245,6 +1246,11 @@ public void onClick(View v) {
                     public void onClick(DialogInterface dialog, int which) {
                         String entry = input.getText().toString();
                         if (!TextUtils.isEmpty(entry) && !mEditingList.contains(entry)) {
+                            if (mEditingList == mSiteSettings.getJetpackWhitelistKeys() && !isValidIpOrRange(entry)) {
```

Comment:
I am not sure why we need the `mEditingList == mSiteSettings.getJetpackWhitelistKeys()` check here. Should we add a comment for why it's needed?